creme/geolocation/bricks.py

```diff
@@ -91,30 +107,57 @@ def detailview_display(self, context):
             context,
             addresses=addresses,
             geoaddresses=addresses,
-            google_api_key=get_google_api_key(),  # TODO: factorise
         ))
 
 
-class GoogleFilteredMapBrick(_MapBrick):
-    id_           = Brick.generate_id('geolocation', 'filtered_google_maps')
-    verbose_name  = _('Filtered addresses on Google Maps ®')
-    template_name = 'geolocation/bricks/google/filtered-map.html'
+class GoogleDetailMapBrick(_DetailMapBrick):
+    id_           = Brick.generate_id('geolocation', 'detail_google_maps')
+    verbose_name  = _('Addresses on Google Maps ®')
+    template_name = 'geolocation/bricks/google/detail-map.html'
+
+    def get_api_key(self):
+        return get_google_api_key()
+
 
+class OpenStreetMapDetailMapBrick(_DetailMapBrick):
+    id_           = Brick.generate_id('geolocation', 'detail_openstreetmap')
+    verbose_name  = _('Addresses on OpenStreetMap ®')
+    template_name = 'geolocation/bricks/leaflet/detail-map.html'
```

Comment:
I find that 'geolocation/bricks/osm/detail-map.html' would be more consistent. Or 'geolocation/bricks/osm/leaflet/detail-map.html', 'geolocation/bricks/osm-leaflet/detail-map.html'. Nope ?

Reply:
'osm' should be fine